### PR TITLE
Add crypto DD commands

### PR DESCRIPTION
**Pull request by minhhoang1023**
Adding `gh` (github_activity) and `news` command

image

image

# Description

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context. 
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes. 
* Provide instructions so we can reproduce. 
* Please also list any relevant details for your test configuration.

# Checklist:

- [ ] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [ ] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

File: openbb_terminal/cryptocurrency/due_diligence/dd_controller.py [hunk 1]
```diff
@@ -1285,3 +1296,162 @@ def call_mcapdom(self, other_args: List[str]):
...
+        )
+
+        parser.add_argument(
+            "-k",
+            "--kind",
+            dest="kind",
+            type=str,
+            help="Filter by category of news. Available values: news or media.",
+            default="news",
+            choices=cryptopanic_model.CATEGORIES,
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/due_diligence/dd_controller.py:1391]: missing autocompletion

File: openbb_terminal/cryptocurrency/due_diligence/dd_controller.py [hunk 2]
```diff
@@ -1285,3 +1296,162 @@ def call_mcapdom(self, other_args: List[str]):
...
+
+        parser.add_argument(
+            "-f",
+            "--filter",
+            dest="filter",
+            type=str,
+            help="Filter by kind of news. One from list: rising|hot|bullish|bearish|important|saved|lol",
+            default=None,
+            required=False,
+            choices=cryptopanic_model.FILTERS,
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/due_diligence/dd_controller.py:1402]: missing autocompletion

File: openbb_terminal/cryptocurrency/due_diligence/dd_controller.py [hunk 3]
```diff
@@ -1285,3 +1296,162 @@ def call_mcapdom(self, other_args: List[str]):
...
+
+        parser.add_argument(
+            "-r",
+            "--region",
+            dest="region",
+            type=str,
+            help="Filter news by regions. Available regions are: en (English), de (Deutsch), nl (Dutch), es (Español), "
+            "fr (Français), it (Italiano), pt (Português), ru (Русский)",
+            default="en",
+            choices=cryptopanic_model.REGIONS,
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/due_diligence/dd_controller.py:1413]: missing autocompletion

File: openbb_terminal/cryptocurrency/due_diligence/dd_controller.py [hunk 4]
```diff
@@ -1285,3 +1296,162 @@ def call_mcapdom(self, other_args: List[str]):
...
+        )
+
+        parser.add_argument(
+            "-s",
+            "--sort",
+            dest="sortby",
+            type=str,
+            help="Sort by given column. Default: published_at",
+            default="published_at",
+            choices=cryptopanic_model.SORT_FILTERS,
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/due_diligence/dd_controller.py:1423]: missing autocompletion

**Review by jose-donato — CHANGES_REQUESTED** [review]
image

What does this activity count mean? number of commits? Maybe we can clarify on the chart?

image
Just a detail but can we prettify the column names? How to do it [here](https://github.com/OpenBB-finance/OpenBBTerminal/blob/e1fd37a5273b2307b3a8e3eb3e117a9b1375b57e/openbb_terminal/cryptocurrency/due_diligence/messari_model.py#L362)


Apart from these small details awesome work as always 🚀

**Review by piiq — CHANGES_REQUESTED** [review]
This looks very nice
Can you also add these commands to a test***.openbb file in the scripts folder?

**Comment by minhhoang1023** [comment]
@piiq @jose-donato Was looking for this detailed level of review. Thanks guys 🚀 All done!

**Review by piiq — APPROVED** [review]
🚀

**Review by jose-donato — APPROVED** [review]
🚀

**Pull request merged**